pennylane_lightning/lightning_gpu/lightning_gpu.toml

```diff
@@ -58,7 +58,8 @@ MultiControlledX       = {}
 # Gates which should be translated to QubitUnitary
 [operators.gates.matrix]
 
-
+BlockEncode            = {properties = [ "controllable" ]}
```

Comment:
Oh do the attributes actually work for gates in the matrix section? 😮

Reply:
How did you test? All controllable instances are already decomposed to matrix so this wouldn't be observable directly.

Reply:
hmm.. I did test the attribute by checking the device specification validator in Catalyst. Removing this line will result the LQ device incompatible supported operations error from the validator in Catalyst.

Reply:
Coming back to this, we did test whether this worked for decomposition, which it doesn't. However it does allow satisfying the verifier. I think in the meantime we can add both
```
BlockEncode            = { properties = [ "controllable" ] }
GlobalPhase             = { properties = [ "controllable" ] }
```
to satisfy the verifier, allowing us to remove this patch as well in Catalyst https://github.com/PennyLaneAI/catalyst/blob/de712cfd803d1a535a009a4a7edb38c898c7aa54/frontend/catalyst/device/qjit_device.py#L495-L499